model/godleyTableWindow.cc

```diff
@@ -1116,7 +1116,7 @@ namespace {
   {	    
     CairoSave cs(cairo);
     int idx=0;
-    if (!godleyIcon.table.multipleEquities || rowCol==row) {  // no column widgets on equity column in single equity column mode
+    if ((rowCol==row || !cminsky().multipleEquities) || (!cminsky().multipleEquities && godleyIcon.table.singleEquity())) {  // no column widgets on equity column in single equity column mode
```

Comment:
The way this conditional is written, the final singleEquity() call is not done due to boolean shortcuts.
I suspect what is needed is:
rowCol==row || !cminsky().multipleEquities || godleyIcon.table.singleEquity()

Reply:
I've tried several combinations, and what I already have comes the closest to making the feature perfect. What I haven't considered is to separate the rows and cols, which may be the way to go...